components/ids-data-publisher/src/main/java/com/wso2telco/ids/datapublisher/util/DataPublisherUtil.java

```diff
@@ -353,6 +353,10 @@ public static void publishTokenEndpointData(Map<String, String> tokenMap) {
             tokenEndpointData.add(tokenMap.get("sessionId"));
         } else {
             tokenEndpointData.add(null);
+        }if (tokenMap.get("status") != null && !tokenMap.get("status").isEmpty()) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "status" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS1192)